resources/scheduled_task.rb

```diff
@@ -17,9 +17,10 @@
 # limitations under the License.
 #
 
-chef_version_for_provides '< 16.0' if respond_to?(:chef_version_for_provides)
+chef_version_for_provides '< 18.0' if respond_to?(:chef_version_for_provides)
```

Comment:
@gholtiii this was added so that the resource shipped in the core of chef infra client would be used over this resource.

https://github.com/chef/chef/blob/master/lib/chef/resource/chef_client_scheduled_task.rb

In other words, these changes should be made in https://github.com/chef/chef

Reply:
Submitted to the core https://github.com/chef/chef/pull/11894

I'll fix this version constraint and resubmit, though it seems this cookbook should be deprecated given the above.